Dockerfile

```diff
@@ -1,4 +1,19 @@
-FROM golang:latest as build
+FROM node:16 as ui
+WORKDIR /build
+
+COPY .git ./.git
+COPY Makefile ./Makefile
+
+# download yarn dependencies
+COPY ui/yarn.lock ./ui/yarn.lock
+COPY ui/package.json ./ui/package.json
+RUN make yarn
```

Comment:
It looks like this is failing on github actions right now. 

https://github.com/pomerium/pomerium/runs/5098456117?check_suite_focus=true

Reply:
This might be network related?
```
 > [linux/arm64 ui 7/9] RUN make yarn:
#32 10.67 [1/4] Resolving packages...
#32 18.73 [2/4] Fetching packages...
#32 103.7 info There appears to be trouble with your network connection. Retrying...
#32 137.3 info There appears to be trouble with your network connection. Retrying...
#32 170.9 info There appears to be trouble with your network connection. Retrying...
#32 234.9 info There appears to be trouble with your network connection. Retrying...
#32 274.1 error An unexpected error occurred: "https://registry.yarnpkg.com/@mui/material/-/material-5.4.0.tgz: ESOCKETTIMEDOUT".
#32 274.1 info If you think this is a bug, please open a bug report with the information provided in "/build/ui/yarn-error.log".
#32 274.1 info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
#32 298.9 make: *** [Makefile:139: yarn] Error 1
------
Dockerfile:10
--------------------
   8 |     COPY ui/yarn.lock ./ui/yarn.lock
   9 |     COPY ui/package.json ./ui/package.json
  10 | >>> RUN make yarn
  11 |     
  12 |     # build ui
--------------------
error: failed to solve: process "/bin/sh -c make yarn" did not complete successfully: exit code: 2
```

Possibly related? 
https://github.com/yarnpkg/yarn/issues/8242